### PR TITLE
Add "produces 750 Energy" to description of Prude.

### PR DESCRIPTION
**Pull request by ecraven**
Add "Produces 750 Energy" to the second tooltip line.

### CODE REVIEW

**Comment by Beherith** [comment]
"Hardened"?

**Comment by WatchTheFort** [comment]
The issue brought up here will be addressed by the eventual new unit info UI, so the name should be left as-is.

**Comment by ecraven** [comment]
One of the reasons for this PR is that the actual energy generation is not shown for this (but it *is* shown for all others). It would be nice for newer players to show how much energy this produces.

File: luaui/Widgets/gui_healthbars.lua [hunk 1]
```diff
@@ -0,0 +1,1480 @@
+function widget:GetInfo()
```
[the remark this answers is not in the document]
Comment (ecraven) [reply luaui/Widgets/gui_healthbars.lua:1]: ah, sorry, no idea how that got in here... the entire file should not be part of this..

[the remark this answers is not in the document]
Comment (ecraven) [reply luaui/Widgets/gui_healthbars.lua]: removed it ;)

**Comment by WatchTheFort** [comment]
The name should be left as Prude, but feel free to add the energy production to the description

**Pull request merged**